entries/s/sidequestvr.com.json

```
@@ -0,0 +1,14 @@
+{
+  "SideQuest": {
+    "domain": "sidequestvr.com",
+    "url": "https://sidequestvr.com/",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://sidequestvr.com/account/security",
+    "categories": [
+      "developer"
```

```suggestion
      "developer",
      "gaming"
```

It seems like SideQuest is a gaming platform as well.

---

I'm not entirely clear. I created an account with this site years ago and haven't really used it since. 😬 I thought it was just a place for developers to easily upload APKs for their Oculus VR apps, but it looks like they do have like a full app store now. 🤷‍♂️